### PR TITLE
make separate yaml for block devices

### PR DESCRIPTION
**Pull request by smruti77**
in xfstests test make separate yaml for block device.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
JOB ID     : b0b713799b3d0aef1dc441204d06a82ac2b700b4
JOB LOG    : /root/vaishnavi/avocado-fvt-wrapper/results/job-2020-08-20T01.46-b0b7137/job.log
 (1/2) xfstests.py:Xfstests.test;run-setup-disk_type-fs_type-fs_ext4-ab37: PASS (407.82 s)
 (2/2) xfstests.py:Xfstests.test;run-setup-disk_type-fs_type-fs_xfs-d140: FAIL: One or more tests failed. Please check the logs. (1628.42 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/vaishnavi/avocado-fvt-wrapper/results/job-2020-08-20T01.46-b0b7137/results.html
JOB TIME   : 2036.51 s

File: fs/xfstests.py.data/xfstests_disk.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+skip_dangerous: True
+scratch_mnt: '/mnt/scratch'
+test_mnt: '/mnt/test'
+disk_mnt: '/mnt/loop-device'
```
[the remark this answers is not in the document]
Comment (smruti77) [reply fs/xfstests.py.data/xfstests_disk.yaml:4]: "disk_mnt: " this is not needed other two are needed.

**Comment by narasimhan-v** [comment]
@smruti77 Can you please upload job.log ?

**Comment by smruti77** [comment]
[xfstests.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/5117052/xfstests.log)

**Comment by narasimhan-v** [comment]
@smruti77 

Does this mean the tests are not run on the given disk ?
Are we missing something ?

```
2020-08-24 00:24:43,052 process          L0605 INFO | Running './check ext4/073'
2020-08-24 00:24:43,334 process          L0437 DEBUG| [stdout] ext4/073 - unknown test, ignored
2020-08-24 00:24:43,362 process          L0437 DEBUG| [stdout] FSTYP         -- ext4
2020-08-24 00:24:43,367 process          L0437 DEBUG| [stdout] PLATFORM      -- Linux/ppc64le ltczz401-lp11 4.18.0-147.el8.ppc64le #1 SMP Thu Sep 26 15:44:34 UTC 2019
2020-08-24 00:24:43,368 process          L0437 DEBUG| [stdout] MKFS_OPTIONS  -- /dev/loop1
2020-08-24 00:24:43,369 process          L0437 DEBUG| [stdout] MOUNT_OPTIONS -- -o acl,user_xattr -o context=system_u:object_r:root_t:s0 /dev/loop1 /mnt/scratch
2020-08-24 00:24:43,369 process          L0437 DEBUG| [stdout] 
2020-08-24 00:24:44,608 process          L0437 DEBUG| [stdout] Passed all 0 tests
2020-08-24 00:24:44,611 process          L0437 DEBUG| [stdout] 
```

**Comment by smruti77** [comment]
> @smruti77
> 
> Does this mean the tests are not run on the given disk ?
> Are we missing something ?
> 
> ```
> 2020-08-24 00:24:43,052 process          L0605 INFO | Running './check ext4/073'
> 2020-08-24 00:24:43,334 process          L0437 DEBUG| [stdout] ext4/073 - unknown test, ignored
> 2020-08-24 00:24:43,362 process          L0437 DEBUG| [stdout] FSTYP         -- ext4
> 2020-08-24 00:24:43,367 process          L0437 DEBUG| [stdout] PLATFORM      -- Linux/ppc64le ltczz401-lp11 4.18.0-147.el8.ppc64le #1 SMP Thu Sep 26 15:44:34 UTC 2019
> 2020-08-24 00:24:43,368 process          L0437 DEBUG| [stdout] MKFS_OPTIONS  -- /dev/loop1
> 2020-08-24 00:24:43,369 process          L0437 DEBUG| [stdout] MOUNT_OPTIONS -- -o acl,user_xattr -o context=system_u:object_r:root_t:s0 /dev/loop1 /mnt/scratch
> 2020-08-24 00:24:43,369 process          L0437 DEBUG| [stdout] 
> 2020-08-24 00:24:44,608 process          L0437 DEBUG| [stdout] Passed all 0 tests
> 2020-08-24 00:24:44,611 process          L0437 DEBUG| [stdout] 
> ```
its running fine, its failing for "unknown test".

**Review by narasimhan-v — APPROVED** [review]

**Pull request merged**